src-java/base-topology/base-messaging/src/test/java/org/openkilda/messaging/info/switches/v2/LogicalPortsValidationEntryV2Test.java

```diff
@@ -147,6 +164,22 @@ public void splitAndUniteManyEntriesLogicalPortEntryTest() {
         assertEquals(entry, united);
     }
 
+    @Test
+    public void splitAndUniteManyEntriesLogicalPortEntryWithErrorMessageTest() {
+        LogicalPortsValidationEntryV2 entry = LogicalPortsValidationEntryV2.builder()
+                .asExpected(true)
+                .error("Some error message")
+                .missing(buildLogicalPortsInfo(1, 500))
+                .excess(buildLogicalPortsInfo(1000, 600))
+                .proper(buildLogicalPortsInfo(2000, 700))
+                .misconfigured(buildMisconfiguredLogicalPortsInfo(3000, 800))
+                .build();
+        List<LogicalPortsValidationEntryV2> list = entry.split(100, 200);
+        assertEquals(14, list.size());
```

Comment:
Could add a message to the assertion explaining what `14` means?

Reply:
thank you, fixed